### PR TITLE
[WIP] Adding Instant Type

### PR DESCRIPTION
**Pull request by ojha-ravi**

### CODE REVIEW

File: helloworld.proto [hunk 1]
```diff
@@ -41,9 +41,16 @@ service Greeter {
...
+  Instant dob = 3;
 }
 
 // The response message containing the greetings
 message HelloReply {
   string message = 1;
 }
+
+message Instant {
+  bool set = 1;
```
Comment (myitcv) [inline helloworld.proto]: This is going to go....

File: src/custom_types.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
+const MAX_32_NUM = Math.pow(2, 31);
```
Comment (myitcv) [inline src/custom_types.ts]: This file can go...

**Comment by ojha-ravi** [comment]
Hey @myitcv , can we take a look at this?

**Comment by ojha-ravi** [comment]
Hey @myitcv , Can we take a look at this the last time??

**Comment by myitcv** [comment]
Closing as we've covered this elsewhere

**Pull request closed without merging**